.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,81 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+    branches:
+      - "*.x"
+      - "master"
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit:
+    name: "PHPUnit"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP with PCOV"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "xdebug"
```

Comment:
"Install PHP with PCOV"

Either wrong label or wrong extension.

Reply:
Good catch!  I had to switch from PCOV to xdebug because of the PHPUnit version but apparently missed the step name. Will fix this tomorrow.

Reply:
On other repositories that support 7.1, I install XDebug for 7.1, and PCOV for the rest, see https://github.com/doctrine/collections/blob/1.6.x/.github/workflows/continuous-integration.yml

Reply:
@greg0ire the issue I ran into right now is the PHPUnit version, which I did not want to upgrade in this PR.